src/main/java/com/sap/oss/phosphor/fosstars/model/value/SecurityReviews.java

```diff
@@ -0,0 +1,143 @@
+package com.sap.oss.phosphor.fosstars.model.value;
+
+import static java.util.Arrays.asList;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * A set of security reviews.
+ */
+public class SecurityReviews implements Set<SecurityReview> {
+
+  /**
+   * Security reviews.
+   */
+  private final Set<SecurityReview> elements = new HashSet<>();
+
+  /**
+   * Create an empty set of security reviews.
+   */
+  public SecurityReviews() {
+
+  }
+
+  /**
+   * Create a set of security reviews.
+   *
+   * @param elements Reviews to be added to the new set.
+   */
+  public SecurityReviews(SecurityReview... elements) {
+    Objects.requireNonNull(elements, "Elements can't be null!");
+    this.elements.addAll(asList(elements));
+  }
+
+  /**
+   * Returns a set of security reviews.
+   *
+   * @return A set of security reviews.
+   */
+  @JsonGetter("elements")
+  public Set<SecurityReview> elements() {
+    return new HashSet<>(elements);
```

Comment:
```suggestion
    return elements;
```

Reply:
This is necessary to make sure that the set can't be implicitly modified by someone.